docs/src/examples/modules/Modal/Usage/ModalExampleCallbacks.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import { Button, Grid, Label, Modal, Segment } from 'semantic-ui-react'
+
+function exampleReducer(state, action) {
+  switch (action.type) {
+    case 'CLEAR_LOG':
+      return { ...state, log: [] }
+    case 'OPEN_MODAL':
+      return {
+        log: [
+          {
```

Comment:
This will add the newest logs in the beginning of the array, is it intentional?

Reply:
Yep, as it creates an order for entries output:

![image](https://user-images.githubusercontent.com/14183168/89026337-d2fa6800-d328-11ea-86ad-9c42c63e3774.png)